app/timetables/models.py

```diff
@@ -1,3 +1,16 @@
 from __future__ import unicode_literals
 
 from django.db import models
+
+from app.shared.models import CommonInfo
+
+
+class Timetable(CommonInfo):
+    name = models.CharField(max_length=60)
```

Comment:
One more thing, sorry ...  please let us make the name field unique, so we will not have duplicate entries in the database.